.github/workflows/main.yml

```diff
@@ -73,7 +51,7 @@ jobs:
 
       - name: Install dependencies only needed for python 3
         if: ${{ matrix.python-version != '2.7' }}
-        run: pip install pandas pytype toml
+        run: pip install pandas pytype toml wrapt
 
       - name: Install common dependencies for Python ${{matrix.python-version}}
         run: pip install future mock pytest-coverage pytest-mock
```

Comment:
Can we try to limit below pytest covery for the python files we changed from py2 to py3 in this feature branch?
```
 93       - name: Run Pytest for python 3 and get code coverage for Codecov
 94         if: ${{ matrix.python-version != '2.7' }}
 95         run: >
 96           pytest
 97           --cov=scripts --cov=ocaml/xcp-rrdd --cov=python3/
 98           scripts/ ocaml/xcp-rrdd python3/ -vv -rA
 99           --junitxml=.git/pytest${{matrix.python-version}}.xml
100           --cov-report term-missing
101           --cov-report xml:.git/coverage${{matrix.python-version}}.xml
102         env:
103           PYTHONDEVMODE: yes

```

Reply:
I can do this.
cc @psafont

Reply:
Had a discussion with @stephenchengCloud , to avoid such kind of interference for the py2->py3 work in this feature branch, we think we may temporarily limit the pytest coverage to only those py files updated from py2 to py3 in this branch. And finally before merging this feature branch back into master, we will resolve all the issues one by one and will finally change the pytest coverage back to "all the py3 files".

Reply:
This idea is fine with me